src/ralph/ralph2_sync/subscribers.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import logging
+from functools import wraps
+
+import pyhermes
+from django.contrib.contenttypes.models import ContentType
+from django.db import transaction
+
+from ralph.assets.models import AssetModel, Environment, ServiceEnvironment
+from ralph.data_center.models import DataCenterAsset
+from ralph.data_importer.models import ImportedObjects
+
+logger = logging.getLogger(__name__)
+
+model_mapping = {
+    'Asset': DataCenterAsset,
+    'AssetModel': AssetModel,
+}
+
+
+class sync_subscriber(pyhermes.subscriber):
+    """
+    Log additional exception when sync has failed.
+    """
+    def _get_wrapper(self, func):
+        @wraps(func)
+        @transaction.atomic
+        def exception_wrapper(*args, **kwargs):
+            try:
+                return func(*args, **kwargs)
+            except:
+                logger.exception('Exception during syncing')
```

Comment:
Should we log here other important information?
I see here only: `'Exception during syncing'`